packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js

```diff
@@ -16,13 +30,48 @@ const getButtonProps = props => {
           { includeSeconds: true }
         )} ago`,
         active: true,
+        exitButton: false,
+        hoverable: true,
+      }
+    }
+    case `SUCCESS`: {
+      return {
+        tooltipContent: (
+          <BuildSuccessTooltipContent
+            isOnPrettyUrl={isOnPrettyUrl}
+            sitePrefix={sitePrefix}
+            buildId={buildId}
+            siteId={siteId}
+            orgId={orgId}
+          />
+        ),
+        active: true,
+        showInfo: true,
```

Comment:
Do we show an exit button everytime `showInfo` is `true`? If that's the case then we don't need both props. Do we need to introduce a new prop for the exit button or can we handle it another way? Even adding it directly onto the `BuildSuccessTooltipContent` could work.